taskfiles/utils.yml

```
@@ -103,3 +103,47 @@ tasks:
         . "{{.OUTPUT_DIR}}/bin/activate"
         pip3 install --upgrade pip
         pip3 install --upgrade -r "{{.REQUIREMENTS_FILE}}"
+
+  # ===
+  # C++ LINTING UTILS
+  # ===
+
+  # @param {string} FLAGS - The flags to pass to clang-format
+  # @param {string} LINT_VENV_DIR - Directory with venv fo linting
+  # @param {[]string} SRC_DIRS - The directories to start linting on.
+  # clang-format.
```

```suggestion
  # Runs clang-format on C++ files in the given paths.
  #
  # @param {string} FLAGS Any flags to pass to clang-format.
  # @param {string} LINT_VENV_DIR Python virtual environment where clang-format is installed.
  # @param {[]string} SRC_DIRS The paths on which to run clang-format.
```

* How about `VENV_DIR`?
* How about `SRC_PATHS`?